openstack/networking/v2/extensions/rbac/requests.go

```diff
@@ -0,0 +1,39 @@
+package rbac
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToRbacCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts represents options used to create a rbac_policy.
+type CreateOpts struct {
```

Comment:
Are any of these options required? If so, can you change the tag to `json:"foo" required:"true"` ?

Additionally, is it acceptable to pass an empty string? If not, the tag should be `json:"foo,omitempty"`.

Reply:
Since all the option fields are required I have added required:"true"